gcp_iam_roles/dynatrace-gcp-function-deployment-role.yaml

```diff
@@ -4,26 +4,24 @@ title: Dynatrace GCP Function deployment role
 description: Role for Dynatrace GCP function helm and pubsub deployment
 stage: GA
 includedPermissions:
-  - pubsub.subscriptions.get
-  - pubsub.subscriptions.list
-  - pubsub.subscriptions.create
-  - pubsub.topics.list
-  - pubsub.topics.create
-  - pubsub.topics.update
-  - pubsub.topics.getIamPolicy
-  - pubsub.topics.setIamPolicy
-  - pubsub.topics.attachSubscription
   - container.clusters.get
   - container.configMaps.create
   - container.configMaps.delete
   - container.configMaps.get
+  - container.configMaps.update
```

Comment:
I don't understand why those changes are her? This PR is about "helm wait"?

Reply:
helm wait needs sample of additional requirements to check state of each components ex. replicasets that are not added before. This file is damp of our role that is working right now (this is not added automatically)

Reply:
Is this written somewhere?

Reply:
but `container.configMaps.update` permission is needed in case when some rubbish are left from previous e2e execution - maybe after changing delete to uninstall (on master branch) this permission is not needed?

But I'm also wondering - is this permission needed for the client to run upgrade helm deployment?

Reply:
to do upgrade is needed

Reply:
all update is not needed if we install fresh new each time